08-cloudwatch-logs/8.1.2.yml

```
@@ -3,6 +3,10 @@ Parameters:
   UbuntuAMI:
     Type: "AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>"
     Default: '/aws/service/canonical/ubuntu/server/20.04/stable/current/amd64/hvm/ebs-gp2/ami-id'
+  VPCID:
+    Type: String
```

Recommend using a stronger type constraint
```suggestion
    Type: 'AWS::EC2::VPC::Id'
```